classes/Guards/CapGuard.php

```
@@ -0,0 +1,53 @@
+	}
+
+	public function run(): void
+	{
+		$data = [
+			'secret' => static::secretKey(),
+			'response' => SubmissionPage::valueFromBody('cap-token')
+		];
+
+		$remote = Remote::post(DreamForm::option('guards.cap.endpoint') . 'siteverify', [
+			'data' => $data
+		]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Kirby\Http\Remote' in method 'run'. (Clean Code Rules)

(StaticAccess)